fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/MatcherHelper.kt

```diff
@@ -85,14 +126,105 @@ object MatcherHelper {
         }
     }
 
+    fun assertItemContainingTextIsGone(vararg appItems: UiObject) {
+        for (appItem in appItems) {
+            assertTrue("${appItem.selector} is not gone", appItem.waitUntilGone(waitingTime))
+            Log.i(TAG, "assertItemContainingTextIsGone: Verified ${appItem.selector} is gone")
+        }
+    }
+
+    fun assertItemTextEquals(vararg appItems: UiObject, itemText: String, shouldTextEqual: Boolean = true) {
```

Comment:
Just a small re-wording suggestion for easier reading: 
itemText -> expectedText
shouldTextEqual -> isEqual

Reply:
Made the change, thanks for the suggestion .